### ISSUE TITLE
Curating the curated patterns

### ISSUE THREAD

**Issue by richtabor** [WordPress/pattern-directory#587]
Can we remove these patterns below from the "Curated" filter on the Patterns Directory, while still having them sent as core patterns within WordPress? 

These aren't quite showcasing the best WordPress has to offer design wise. 

### Patterns
https://wordpress.org/patterns/pattern/text-only-header-with-tagline/
https://wordpress.org/patterns/pattern/simple-header-with-tagline/
https://wordpress.org/patterns/pattern/site-title-and-menu-button/
https://wordpress.org/patterns/pattern/centered-header-with-logo/
https://wordpress.org/patterns/pattern/header-with-large-font-size/
https://wordpress.org/patterns/pattern/simple-header/
https://wordpress.org/patterns/pattern/centered-footer/
https://wordpress.org/patterns/pattern/footer-with-search-site-title-and-credit-line/
https://wordpress.org/patterns/pattern/footer-with-site-title-and-credit-line/
https://wordpress.org/patterns/pattern/footer-with-navigation-and-credit-line/
https://wordpress.org/patterns/pattern/centered-footer-with-social-links/
https://wordpress.org/patterns/pattern/footer-with-credit-line-and-navigation/
https://wordpress.org/patterns/pattern/left-aligned-footer/
https://wordpress.org/patterns/pattern/three-columns-of-text/
https://wordpress.org/patterns/pattern/media-text-in-a-full-height-container/
https://wordpress.org/patterns/pattern/media-text-with-image-on-the-right/
https://wordpress.org/patterns/pattern/heading/
https://wordpress.org/patterns/pattern/quote/
https://wordpress.org/patterns/pattern/two-columns-of-text/
https://wordpress.org/patterns/pattern/two-images-side-by-side/

**Comment by richtabor** [comment]
cc @ryelle

**Comment by ryelle** [comment]
> Can we remove these patterns below from the "Curated" filter on the Patterns Directory, while still having them sent as core patterns within WordPress?

If we remove the `core` tag and add the "Featured" category, they should be sent to core without being in the curated section on the directory. But they'll show up in the "Featured" category on PD and in WP.

In 580, we decided [that the patterns sent to core and the curated patterns are the same](https://github.com/WordPress/pattern-directory/issues/580#issuecomment-1552207703), so that's the logic used to show "curated" patterns (that `core` tag). So if we remove the core tag, they won't be "curated" anymore, but they won't be sent to core either. The "Featured" suggestion is a workaround.

**Comment by richtabor** [comment]
> So if we remove the core tag, they won't be "curated" anymore, but they won't be sent to core either. The "Featured" suggestion is a workaround.

I think that's fine. Some of these are super basic, and I think it's fine to reserve the "Featured" category for local patterns — not the directory's featured patterns. 

That saying, can we get [these four patterns ](https://wordpress.org/patterns/categories/featured/)removed from the "Featured" category?

**Comment by richtabor** [comment]
> I think that's fine. Some of these are super basic, and I think it's fine to reserve the "Featured" category for local patterns — not the directory's featured patterns.

Here's an issue on removing core patterns from the featured category: https://github.com/WordPress/pattern-directory/issues/612

**Comment by richtabor** [comment]
Following up on this. How can we move this forward?

**Comment by ryelle** [comment]
It sounds like it's still unclear what to do here— you wanted these removed from curated, but still sent to core WP sites. I suggested we can use the "Featured" category workaround, which will remove these from curated but send to sites, _but_ it will add them to the Featured category in the sites, which it sounds like you don't want either.

Those are the two ways patterns are sent to WP sites:

1. They're marked with the core keyword == ["Curated"](https://wordpress.org/patterns/)
2. They're added to the Featured category == ["Featured"](https://wordpress.org/patterns/categories/featured/?curation=all)

One thing I might be able to do is removed `featured` from the API response sent to WP sites, if you know you will _never_ want pattern-directory patterns in the featured category (which would include patterns loaded via slugs in theme.json). Then "featured" on wporg/patterns and "featured" on local WP sites would be fully separate things, and remote patterns would never be in featured on local sites.

**Comment by richtabor** [comment]
> One thing I might be able to do is removed featured from the API response sent to WP sites, if you know you will never want pattern-directory patterns in the featured category (which would include patterns loaded via slugs in theme.json). Then "featured" on wporg/patterns and "featured" on local WP sites would be fully separate things, and remote patterns would never be in featured on local sites.

Yes, I think that no patterns from the directory should be assumed as "featured" within the Inserter. We shouldn't guess what patterns are most interesting (i.e. featured) for sites. If a theme includes a pattern locally and it's set to the featured category, that's fine. 

Would be neat if a theme can set "featured" patterns from directory patterns, but I don't want that (with all its unknowns) to block this. Maybe that's something with the theme.json registration of patterns.

**Comment by ryelle** [comment]
@richtabor I'm doing some triage here (with the hope of some dedicated dev time in the new year), and I'm just trying to figure out where we landed here. In #704, code was added to prevent "Featured" patterns from being in the "Featured" category on sites, but "Featured" patterns are still sent to core. So the patterns you've listed above that are currently "Curated" would be removed from "Curated" and added to "Featured" — they'll appear on client sites, but not appear in the "Curated" filter on the Pattern Directory. Is that still what you want to happen?

> That saying, can we get [these four patterns](https://wordpress.org/patterns/categories/featured/) removed from the "Featured" category?

Did #704 address the issue here, or do these still need an update?
